src/old_ui/Icon/Icon.jsx

```diff
@@ -22,7 +22,10 @@ function Icon({
   if (!IconSvg) return null
   return (
     <span className={cs(color, className)} data-testid={testId}>
-      <IconSvg className={cs(iconClass ? iconClass : IconClasses[size])} />
+      <IconSvg
+        className={cs(iconClass ? iconClass : IconClasses[size])}
+        data-testid={`${name}-svg`}
```

Comment:
this is a massive W

Reply:
So with Vite + Vitest, it actually inlines the svg's so our old way of trying to grab them with `findByText('icon.svg)` just doesn't work hah